Frontend/CO.CDP.OrganisationApp/Constants/ApiExceptionMapper.cs

```diff
@@ -29,7 +31,7 @@ public static void MapApiExceptions(
             : null;
     }
 
-    private static string GetErrorMessageByCode(string errorCode)
+    private static string GetErrorMessageByCode(string errorCode, string exceptionMessage)
```

Comment:
Doesn't this need to be an optional parameter?

Reply:
Done